generator/internal/pagination.cc

```diff
@@ -84,10 +89,63 @@ DeterminePagination(MethodDescriptor const& method) {
                         std::get<1>(repeated_message_fields[0]));
 }
 
+// For both the sqladmin and compute proto definitions, the paging conventions
+// do not adhere to aip-4233, but the intent is there. If we can make it work,
+// add pagination for any such methods.
+google::cloud::optional<std::pair<std::string, Descriptor const*>>
+DetermineRestPagination(MethodDescriptor const& method) {
```

Comment:
`DetermineRestPagination` seems too optimistic.  Maybe `DetermineAlternativePagination`?  No biggie, as this is in the anonymous namespace anyway.